.github/workflows/valid-style.yml

```diff
@@ -85,5 +85,5 @@ jobs:
           source share/spack/setup-env.sh
           spack debug report
           spack -d bootstrap now --dev
-          spack style -t black
+          spack -d style -t black
```

Comment:
I think it's not, but I also think that we want to keep it at least for now. IIRC we were getting really useless error messages because `spack style` was failing in new and interesting ways, and that's why we added it.